client/js/table-model.js

```diff
@@ -16,6 +16,21 @@ class TableModel {
   setValue(location, value) {
     this.data[this._getCellId(location)] = value;
   }
+
+  getSum(column) {
+    let sum = [];
+    for (var prop in this.data) {
+      if (parseInt(prop[0], 10) === column) {
+        if (!isNaN(parseInt(this.data[prop], 10))) {
+          sum.push(parseInt(this.data[prop], 10));
+        }
+      }
+    }
+    if(sum.length > 1) {
+      sum = sum.reduce((b, a) => b + a);
```

Comment:
Why are `b` and `a` in the opposite of their usual order?  It's good to keep things consistent unless there's a real reason.  Meaningless differences distract your code readers from the point of what you're doing.